pkgs/top-level/all-packages.nix

```diff
@@ -20315,6 +20315,8 @@ in
 
   lbzip2 = callPackage ../tools/compression/lbzip2 { };
 
+  hactool = callPackage ../tools/compression/hactool { };
+
```

Comment:
do you mind trying to order this a little better? its squashed between two packages that start with l.

The package list is already a little in dissarray, but may as well not contribute to it.

Reply:
Moved it